test/fixtures/poro.rb

```diff
@@ -68,6 +68,11 @@ class ProfilePreviewSerializer < ActiveModel::Serializer
   attributes :name
 end
 
+class ProfileTypeSerializer < ActiveModel::Serializer
+  attributes :name
+  type 'profile'
+end
```

Comment:
Haha... for me whole this minitest set is already insane. I agree, so I define it inside of file with test?